app/views/parents/helpers/levelCompletionHelper.js

```diff
@@ -0,0 +1,43 @@
+import Level from '../../../models/Level'
+
+export function getProgressStatusHelper (levelSessions, { slug, fromIntroLevelOriginal }) {
+  let status = 'not-started'
+  if (!levelSessions || (!slug && !fromIntroLevelOriginal)) return status
+  const ls = levelSessions.filter(ls => ls.levelID === slug || (fromIntroLevelOriginal && ls?.level?.original === fromIntroLevelOriginal))
+  if (ls.length) status = 'in-progress'
+  ls.forEach(session => {
+    if (session.state.complete) status = 'complete'
```

Comment:
It seems this will set the status to `complete` if at least one session is complete, but the others maybe not. Is this the intentional behaviour?

Reply:
it is for a level instead of a campaign/course, so I think it's ok if level is completed in any language